broker/lib/controllers/FabrikBaseController.js

```diff
@@ -30,6 +33,71 @@ class FabrikBaseController extends BaseController {
     return _.get(config, 'broker_name', 'service-fabrik-broker');
   }
 
+  handleByLockingResource(func, operationType, lastOperationCall) {
```

Comment:
Minor comment: Can this function be renamed to something more understandable? "Handle what" is my question. Just think over it

Reply:
Changed to `handleWithResourceLocking`